client/v2/client.go

```diff
@@ -101,7 +120,11 @@ func (self *Client) attributesUrl() string {
 	return self.baseUrl + path.Join("attributes")
 }
 
-func (self *Client) httpGetResponse(postData interface{}, url, infoName string) ([]byte, error) {
+func (self *Client) statsUrl(name string) string {
+	return self.baseUrl + path.Join("stats", name)
```

Comment:
Yeah. Thats possible. Updated. I followed the existing templates, which are weird in retrospect.